spl/src/token_interface.rs

```diff
@@ -0,0 +1,71 @@
+use anchor_lang::solana_program::pubkey::Pubkey;
+use std::ops::Deref;
+
+static IDS: [Pubkey; 2] = [spl_token::ID, spl_token_2022::ID];
+
+#[derive(Clone, Debug, Default, PartialEq)]
+pub struct TokenAccount(spl_token_2022::state::Account);
```

Comment:
Does this mean that `TokenAccount` can no longer be used in regular `Account` types? Or is the idea that people import different `TokenAccount`s , token_interface::TokenAccount vs. token::TokenAccount. I just wonder about the naming.

Reply:
Thanks for the clarification, it sounds like a good approach.